Makefile

```
@@ -153,7 +153,7 @@ setup-operator-sdk-run: ## Create ns, crds, sa, role, and rolebinding before ope
 	- . ./scripts/operator_sdk_sa_kubeconfig.sh $(CLUSTER_SERVER) $(NAMESPACE) $(SERVICE_ACCOUNT)
 
 operator-sdk-run: ## Run operator locally outside the cluster during development cycle
-	operator-sdk run local --watch-namespace="" --kubeconfig=./sa.kubeconfig
+	operator-sdk run --local --watch-namespace="" --kubeconfig=./sa.kubeconfig
```

I suggest keeping the original line as `--local` is deprecated in newer versions of operator-sdk.
```suggestion
	operator-sdk run local --watch-namespace="" --kubeconfig=./sa.kubeconfig
```

---

Ah didn't realize that. Kishore just make sure you have the latest operator-sdk.